homeassistant/loader.py

```diff
@@ -1247,6 +1247,18 @@ def __getattr__(self, comp_name: str) -> ModuleWrapper:
         if component is None:
             raise ImportError(f"Unable to load {comp_name}")
 
+        from .helpers.frame import report  # pylint: disable=import-outside-toplevel
+
+        report(
+            (
+                f"accesses hass.components.{comp_name}."
+                " This is deprecated and will stop working in Home Assistant 2024.6, it"
```

Comment:
Lets make it 6 months (https://github.com/home-assistant/developers.home-assistant/pull/2093#pullrequestreview-1906411916)

```suggestion
                " This is deprecated and will stop working in Home Assistant 2024.9, it"
```

Reply:
Was this forgotten?

Reply:
Yikes, yes, thought I've hit "Commit suggestion". Will create a PR.

Reply:
Fixed in https://github.com/home-assistant/core/pull/111903. Thanks for noticing it.